cmd/bcpd/app/app_test.go

```diff
@@ -647,17 +646,19 @@ func benchmarkSendTxWithMultisig(b *testing.B, nbAccounts, blockSize, nbContract
 		c.id = createContract(b, myApp, chainID, int64(i+1), []*account{signer}, threshold, c.sigs()...)
 	}
 
-	txs := make([][]byte, b.N)
+	txs := make([][]byte, b.N*blockSize)
```

Comment:
So we are timing how long it takes to build a block of size X,
rather than what the average time of a transaction, when included in a block of size X (amortizing the read/write cost).

I think the second is nicer to compare.
Rather I would change the loop to send the blocks below...
sending b.N/blockSize blocks (and yes, maybe add one last block to deal with any remainder).